src/util/replaygain.h

```diff
@@ -56,6 +69,18 @@ class ReplayGain {
         m_peak = CSAMPLE_PEAK;
     }
 
+    // Parse and format the peak value metadata according to the ReplayGain
+    // 1.0/2.0 specification.
+    // http://wiki.hydrogenaud.io/index.php?title=ReplayGain_1.0_specification
+    // http://wiki.hydrogenaud.io/index.php?title=ReplayGain_2.0_specification
+    static CSAMPLE parsePeak(QString strPeak, bool* pValid = 0);
```

Comment:
An additional hint that strPeak = "1.0" represents the digital full scale could be useful.